clarifai/client/dataset.py

```diff
@@ -16,7 +32,232 @@ def __init__(self, dataset_id: str, **kwargs):
     """
     self.kwargs = {**kwargs, 'id': dataset_id}
     self.dataset_info = resources_pb2.Dataset(**self.kwargs)
-    super().__init__(user_id=self.user_id, app_id=self.app_id)
+    BaseClient.__init__(self, user_id=self.user_id, app_id=self.app_id)
+    Lister.__init__(self)
+
+  def _upload_inputs(self, batch_input: List[resources_pb2.Input]) -> str:
+    """Upload inputs to clarifai platform dataset.
+    Args:
+      batch_input: input batch protos
+    Returns:
+      input_job_id: Upload Input Job ID
+    """
+    input_job_id = uuid.uuid4().hex  # generate a unique id for this job
+    response = self._grpc_request(self.STUB.PostInputs,
+                                  service_pb2.PostInputsRequest(
+                                      user_app_id=self.user_app_id,
+                                      inputs=batch_input,
+                                      inputs_add_job_id=input_job_id))
+    if response.status.code != status_code_pb2.SUCCESS:
+      try:
+        print(f"Post inputs failed, status: {response.inputs[0].status.details}")
+      except:
+        print(f"Post inputs failed, status: {response.status.details}")
+
+    return input_job_id
+
+  def _upload_annotations(self, batch_annot: List[resources_pb2.Annotation]
+                         ) -> Union[List[resources_pb2.Annotation], List[None]]:
+    """Upload image annotations to clarifai detection dataset
+    Args:
+      batch_annot: annot batch protos
+    Returns:
+      retry_upload: failed annot upload
+    """
+    retry_upload = []  # those that fail to upload are stored for retries
+    response = self._grpc_request(
+        self.STUB.PostAnnotations,
+        service_pb2.PostAnnotationsRequest(user_app_id=self.user_app_id, annotations=batch_annot),
+    )
+    if response.status.code != status_code_pb2.SUCCESS:
+      try:
+        print(f"Post annotations failed, status: {response.annotations[0].status.details}")
```

Comment:
Can this be `logger.info` instead of `print` to maintain consistency throughout the SDK.